contents/handbook/people/team-structure/infrastructure.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Team Infrastructure & Deployments
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+![Image of Cloud Infrastructure](https://github.com/PostHog/posthog-cloud/blob/master/docs/images/infra.png?raw=true)
+
+## People
+
+- [James Greenhill](/handbook/company/team/#james-greenhill-software-engineer) (Team lead, Data/Infra Engineer)
+- [Karl-Aksel Puulmann](/handbook/company/team/#karl-aksel-puulmann-software-engineer) (Full Stack Engineer)
+
+## Mission
+
+Make using and developing for PostHog as reliable as running water. Wherever you want it.
+
+## Goals 
+
+- We don't lose events
+- Data is as up to date as possible
+- Engineers always be able to ship and build
+- Fail fast. Fix faster.
+- Ship anywhere
+- Stack performs at speed of click
+- Support Small Teams (and contributors) in building and debugging Posthog
+
+## Responsibilities
+Concrete things we take responsibility over:
+
+- [app.posthog.com](app.posthog.com) and its infrastructure
+- On Prem & Single Tenant deployments
+- CI/CD - How we deploy
+- Data infrastructure (Clickhouse, Kafka)
+- Monitoring and Alerting stack
+
+## Customer
+
+- Other Small Teams in making sure they have the tools they need to build
+- End users (Teams building products and using Posthog for Product Analytics)
+- Single tenant customers such as VPC deployment users
+
+## Output metrics
+
+### VPC
+###### Retention 
+- Metric: Retention
+- Objective: Better than cloud
+### Cloud
+###### Data Loss
+- Metric: Data loss %
+- Objective: < 0.1%
+###### Uptime 
+- Metric: Uptime
+- Objective: > 99.99%
+###### Speed 
```

Comment:
I don't think this belongs under this team.

IMO Core Analytics should be responsible for query response times and integrations for event ingestion. What this team can do is support them with the infrastructure to build on top of.

Reply:
I think you are right. Main thing is we want to stay ahead of core to make sure that we always have anticipated demand and provisioned enough resources to cope with the demand.